### PR TITLE
Revise content for the Resources quest

### PR DESCRIPTION
**Pull request by kjhenner**

### CODE REVIEW

**Comment by kjhenner** [comment]
This is a significant revision of the quest. I'd appreciate a scan for typos or errors from a reviewer!

File: Quest_Guide/quests/resources.md [hunk 1]
```diff
@@ -124,8 +134,8 @@ likely encounter most often:
 * `exec` An external command
 * `host` A host entry
 
-If you are curious to learn about all of the different built-in resources types
-available for you to manage, see the [Type Reference
+If you are curious to learn about all of the built-in resources types
+available, see the [Type Reference
 Document](http://docs.puppetlabs.com/references/latest/type.html)
```
Comment (casharma) [inline Quest_Guide/quests/resources.md]: callout to `puppet describe --list` would be nice, too.

File: Quest_Guide/quests/resources.md [hunk 2]
```diff
@@ -134,12 +144,28 @@ Take another look at the first line of the resource declaration.
...
-means that no two resources of the same type can share a title. In the case of
-the user resource, the title is also the name of the user account being managed.
+The single quoted word `'root'` just before the colon is the resource **title**.
+Puppet uses the resource title as its own internal unique identifier for that
+resource. This means that no two resources of the same type can have the same title.
+
+In our example, the resource title, `'root'`, is also the name of the user we're inspecting
+with the `puppet resource` command. Generally, a resource title will match the name
+of the thing on the system that the resource is managing. A package resource will
+usually, be titled with the name of the managed package, for example, and a file resource
```
Comment (casharma) [inline Quest_Guide/quests/resources.md]: drop comma after usually

File: Quest_Guide/quests/resources.md [hunk 3]
```diff
@@ -175,17 +201,16 @@ type {'title':
...
+best practice to include it for the sake of consistency. Leave it out, and you'll
+inevitably forget to insert it when you add another attribute value pair on the
+following line!
 {% endaside %}
 
-So in the world of Puppet, you and everything around you can be respresented as
-a resource, and resources follow this tidy declarative syntax. As pretty as they
-are, presumably you don't want to just look at resources all day, you want to
-change them! 
+In the world of Puppet, everything around you can be respresented as
```
Comment (casharma) [inline Quest_Guide/quests/resources.md]: respresented -> represented

**Comment by casharma** [comment]
Done reviewing :+1:

**Comment by kjhenner** [comment]
Amended commit to address those comments.

**Pull request merged**